### PR TITLE
0 intro david tereshchenko

### PR DESCRIPTION
**Pull request by DavidTereshchenko**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,8 +1,12479 @@
 {
```
Comment (vladyslav-yermolin-moc) [inline package-lock.json:1]: Hi, @DavidTereshchenko, please remove this file from your pr

**Review by vladyslav-yermolin-moc — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 1 above)
Comment (DavidTereshchenko) [reply package-lock.json]: ready)

Comment (MZabolotnev) [reply package-lock.json]: You shoud remove this changes, not existing file)

Comment (DavidTereshchenko) [reply package-lock.json:1]: ooops, i fixed it

File: package-lock.json [hunk 2]
```diff
@@ -1,8 +1,12479 @@
...
   "name": "fe-course-basic-2023",
   "version": "21.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```
Comment (vladyslav-yermolin-moc) [inline package-lock.json:4]: here is the change

Comment (vladyslav-yermolin-moc) [reply package-lock.json]: In this task you should only add your Readme, not change the settings of whole repo.
So yes, if you change 2 to 1 back, it returns the file to initial state and remove its changes from your pull-request.

Comment (DavidTereshchenko) [reply package-lock.json]: the most interesting thing is that I didn't touch that file, I don't even know why there are changes)

Comment (DavidTereshchenko) [reply package-lock.json]: did i do my homework ?

Comment (vladyslav-yermolin-moc) [reply package-lock.json]: Hi, there, sorry, but not. You still have some changes for the project's package-lock.json. If I merge your pr, these changes will rewrite it. And that is not, what you have to do in this task.
I've wrote a way to deal with this problem and its explanation in tg chat - https://t.me/c/1602765599/177.
Or you could ask Bard/ChatGPT what's going on and how to solve it

**Pull request closed without merging**